### PR TITLE
About us page tests

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/identifiers/MainSite/pages/company/AboutUs.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export default class CloudDevelopment extends GeneralContainersMainSite {
+    static OurStory = 'Container-OurStory';
+    static WhatsAtTheCore = 'Container-WhatsAtTheCore';
+    static OurTeam = 'Container-OurTeam';
+    static WhatMakesUsSpecial = 'Container-WhatMakesUsSpecial';
+    static OurPartners = 'Container-OurPartners';
+    static ShoutoutFromOurPartners = 'Container-ShoutoutFromOurPartners';
+    static OurPeople = 'Container-OurPeople';
+    static CarouselPhoto = 'CarouselPhoto';
+}
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/MainSite/pages/company/AboutUs.ts:12]: where this variable is used?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		['What makes\nus special', '04'],
+		['Our partners', '05'],
+		['Shoutout from\nour partners', '06'],
+		['Our people', '07'],
+		['Get in Touch', '08'],
+	];
+	
+	await baseDriverSteps.checkContainerTitlesAndNumbers(containers, expectedData);
+});
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:51]: pls add checks for 02 "What’s at the Core" block

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+	]);
+
+	for (const [button, url] of buttonUrlMap.entries()) {
+		await button.click();
+		const newPage = await driver.DriverContext.waitForEvent('page');
+		expect(newPage.url()).toContain(url);
+		await newPage.close();
+	}
+});
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:134]: pls add checks for 04 "What makes us special" container

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+test('Check "Learn more about how we work" button from the "Our partners" block on the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+	const ourPartnersBlock = driver.getByTestId(AboutUs.OurPartners);
+
+	await ourPartnersBlock.getByTestId(MainSiteButtons.Arrow).click();
+	await baseDriverSteps.checkUrl(UrlProvider.urlBuilder(UrlPath.HowWeWork));
+});
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:140]: psl add checks for images

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+	]);
+
+	for (const [button, url] of buttonMap) {
+		await button.click();
+		const newPage = await driver.DriverContext.waitForEvent('page');
+		expect(newPage.url()).toContain(url);
+		await newPage.close();
+	}
+});
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:185]: pls add checks for 07 "Our people" container

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+	const shoutoutFromOurPartnersContainer = driver.getByTestId(AboutUs.ShoutoutFromOurPartners);
+
+	const linkedInButtons = await shoutoutFromOurPartnersContainer.getByTestId(MainSiteButtons.LinkedInReviewArrow).all();
+
+	const linkedInReviewsTab = shoutoutFromOurPartnersContainer.getByTestId(MainSiteButtons.LinkedInReviews);
+	expect(await linkedInReviewsTab.getAttribute('class')).toContain('active');
+
+	const buttonMap = new Map([
+		[linkedInButtons[0], LinkedInReviewLinks.FerdiVanHeerden],
+		//[linkedInButtons[1], LinkedInReviewLinks.GrahamBrown], waiting for fix from https://ts-website.atlassian.net/browse/TSWEB-1040
```
Comment (ysukhova) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:153]: Please use JiraId skip

**Review by ysukhova — CHANGES_REQUESTED** [review]

**Review by ysukhova — APPROVED** [review]

**Review by DaniilVostrikov — APPROVED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

File: ui_tests_playwright/automation/identifiers/Buttons.ts [hunk 7]
```diff
@@ -55,4 +55,6 @@ export default class Buttons {
 	// Other
 	static Logo = 'Logo-Techstack';
 	static Blog = 'Button-Blog';
+	static Carousel_Button_Prev = 'CarouselButtonPrev';
+	static Carousel_Button_Next = 'CarouselButtonNext';
 }
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/Buttons.ts:60]: I think we have these identifiers in Container class

File: ui_tests_playwright/automation/identifiers/MainSite/MainSiteButtons.ts [hunk 8]
```diff
@@ -25,6 +25,7 @@ export default class MainSiteButtons {
 	static ScheduleACall = 'Button-ScheduleACall';
 
 	static LetsMakeItTogether = 'Button-LetsMakeItTogether';
+	static Arrow = 'Arrow';
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/MainSite/MainSiteButtons.ts:29]: This identifier also exist in Container class

File: ui_tests_playwright/automation/identifiers/MainSite/pages/company/AboutUs.ts [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+export default class CloudDevelopment extends GeneralContainersMainSite {
+    static OurStory = 'Container-OurStory';
+    static WhatsAtTheCore = 'Container-WhatsAtTheCore';
+    static OurTeam = 'Container-OurTeam';
+    static WhatMakesUsSpecial = 'Container-WhatMakesUsSpecial';
+    static OurPartners = 'Container-OurPartners';
+    static ShoutoutFromOurPartners = 'Container-ShoutoutFromOurPartners';
+    static OurPeople = 'Container-OurPeople';
+    static CarouselPhoto = 'CarouselPhoto';
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/MainSite/pages/company/AboutUs.ts:11]: mb better add this identifier to Container as it's not container?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+import { LinkedInReviewLinks } from '../../../../preconditionsData/Links/linkedInReviewLinks';
+import { ClutchReviewLinks } from '../../../../preconditionsData/Links/ClutchReviewLinks';
+
+test.beforeEach(async () => {
+	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.AboutUs));
+});
+
+test('Check the header from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+	const info = driver.getByTestId(AboutUs.Info);
+	await expect(info.getByTestId(Container.Title)).toHaveText('We Make an Impact on\nthe Product, People, and\nWorld');
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:23]: mb need to add breadcrumbs text check?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 11]
```diff
@@ -0,0 +1,253 @@
...
+		'Team growth',
+		'Making an impact (CSR)',
+		'Sharing expertise',
+	];
+	await expect(allSectionTitles).toHaveText(testData);
+});
+
+test('Check "Learn more about how we work" button from the "Our partners" block on the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+	const ourPartnersBlock = driver.getByTestId(AboutUs.OurPartners);
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:189]: should we need to add checking images in "Our partners" block?

File: ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts [hunk 12]
```diff
@@ -0,0 +1,253 @@
...
+		await button.click();
+		const newPage = await driver.DriverContext.waitForEvent('page');
+		expect(newPage.url()).toContain(url);
+		await newPage.close();
+	}
+});
+
+test('Check photo carousel from the "Our people" block from the "About Us" page @Regression @AboutUs @TSWEB-1022', async () => {
+	const ourPeopleContainer = driver.getByTestId(AboutUs.OurPeople);
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/company/AboutUs.test.ts:240]: pls add "Join Us" button click

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

**Pull request merged**